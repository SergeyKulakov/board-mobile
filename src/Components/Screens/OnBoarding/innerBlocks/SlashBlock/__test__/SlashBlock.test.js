import React from 'react'
import { shallow } from 'enzyme'
import { expect as chaiExpect } from 'chai'

import 'jest-styled-components'

import { SlashBlock as Component } from '../index'

describe('OnBoarding.SlashBlock', () => {
  const defaultProps = {
    image: 0,
    title: 'title',
    subTitle: 'sub title',
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
