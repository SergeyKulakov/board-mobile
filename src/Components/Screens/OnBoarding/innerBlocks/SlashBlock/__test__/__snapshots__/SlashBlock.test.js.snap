// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnBoarding.SlashBlock render snapshot 1`] = `
<TouchableWithoutFeedback>
  <Styled(View)>
    <Styled(View)>
      <Styled(Image)
        resizeMode="contain"
        source={0}
        testID="image"
      />
    </Styled(View)>
    <Styled(View)>
      <Styled(Styled(Text))
        testID="title"
      >
        title
      </Styled(Styled(Text))>
      <Styled(Styled(Text))
        testID="subTitle"
      >
        sub title
      </Styled(Styled(Text))>
    </Styled(View)>
  </Styled(View)>
</TouchableWithoutFeedback>
`;
